Jenkinsfile

```diff
@@ -82,7 +82,7 @@ def rpm_test_daos_test = '''me=\\\$(whoami)
                             cat /etc/daos/daos_server.yml
                             cat /etc/daos/daos_agent.yml
                             module load mpi/openmpi3-x86_64
```

Comment:
I guess we don't need this either if removing the `orterun`s below, yes?

Reply:
actually, I think you might still need it because daos_test is an MPI application so will need libmpi to be in LD_LIBRARY_PATH if it's not added in rpath